### PR TITLE
STM32: add MASK defines in sdio.h

### PR DESCRIPTION
**Pull request by tsaitgaist**
this also fixes a typo and  renames two MSK to MASK

### CODE REVIEW

File: include/libopencm3/stm32/sdio.h [hunk 1]
```diff
@@ -79,13 +79,14 @@
 /* --- SDIO_POWER values --------------------------------------------------- */
 
 #define SDIO_POWER_PWRCTRL_SHIFT	0
+#define SDIO_POWER_PWRCTRL_MASK		(0x3 << SDIO_POWER_PWRCTRL_SHIFT)
```
Comment (karlp) [inline include/libopencm3/stm32/sdio.h]: For consistency with _most_ other places, when both SHIFT and MASK are defined, the mask is unshifted.

File: include/libopencm3/stm32/sdio.h [hunk 2]
```diff
@@ -95,7 +96,8 @@
 
 /* WIDBUS: Wide bus mode enable bit */
 /* set the width of the data bus */
-#define SDIO_CLKCR_WIDBUS_SHIFT		11
+#define SDIO_CLKCR_WIDBUS_SHIFT	11
+#define SDIO_CLKCR_WIDBUS_MASK	(0x3 << SDIO_CLKCR_WIDBUS_SHIFT)
```
Comment (karlp) [inline include/libopencm3/stm32/sdio.h]: same here

**Comment by ChuckM** [comment]
Yes, please leave the mask itself unshifted which allows for passing around parameters in the smallest possible integer type.

**Comment by tsaitgaist** [comment]
thanks for the comments. I'm closing this PR in favour of the cleaner PR #838

**Pull request closed without merging**